File: mpp/lab3/src/main/java/domain/Participant.java
package domain;

import repository.DatabaseField;
import repository.DatabaseTable;

@DatabaseTable(name="Participants")
public class Participant {

    @DatabaseField(name="ParticipantName", type="nvarchar(255)", primaryKey=true, notNull=true)
    private String name;

    public Participant() {}

    public Participant(String name) {
        this.name = name;
    }

    public String getName() {
        return name;
    }

    @Override
    public String toString() {
        return "Participant{" +
                "name='" + name + '\'' +
                '}';
    }
}

File: mpp/lab5/domain/Participant.cs
namespace lab5.domain
{
    public class Participant
    {
        public int Id { get; set; }
        public string Name { get; }

        public Participant(int id, string name)
        {
            Id = id;
            Name = name;
        }

        public Participant(string name)
                : this(0, name)
        { }

        public override string ToString()
        {
            return $"Participant{{Id={Id}, Name={Name}}}";
        }
    }
}
